mylyn.builds/org.eclipse.mylyn.builds-feature/p2.inf

```diff
@@ -1,5 +1,18 @@
+###############################################################################
+# Copyright (c) 2023 ArSysOp
+# 
+# This program and the accompanying materials are made available under the
+# terms of the Eclipse Public License 2.0 which is available at
+# https://www.eclipse.org/legal/epl-2.0/.
+# 
+# SPDX-License-Identifier: EPL-2.0
+# 
+# Contributors:
+#   Alexander Fedorov (ArSysOp) - initial API and implementation
+###############################################################################
+
 update.matchExp = providedCapabilities.exists(pc | \
    pc.namespace == 'org.eclipse.equinox.p2.iu' && \
      (pc.name == 'org.eclipse.mylyn.builds.feature.group' || \
-       (pc.name == 'org.eclipse.mylyn.builds.feature.feature.group' && pc.version ~= range('[0.0.0,4.0.0)'))))
+       (pc.name == 'org.eclipse.mylyn.builds.feature.feature.group' && pc.version ~= range('[0.0.0,$version$)'))))
```

Comment:
Ed @merks , all the `p2.inf` moved from hardcoded version to `$version$` variable, otherwise they will stop working after feature version increment.

Reply:
Most probably @merks has no time for Mylyn now

Reply:
Sorry, I'm so swamped I could cry.  I didn't notice this...

Reply:
You live in a universe where m1 comes later than in mine!  😱